File: content/single-module-application/src/main/java/${project_rootPath}/security/FullAccessRole.java
package ${project_rootPackage}.security;

import io.jmix.security.model.EntityAttributePolicyAction;
import io.jmix.security.model.EntityPolicyAction;
import io.jmix.security.role.annotation.EntityAttributePolicy;
import io.jmix.security.role.annotation.EntityPolicy;
import io.jmix.security.role.annotation.Role;
import io.jmix.security.role.annotation.SpecificPolicy;
import io.jmix.securityui.role.annotation.MenuPolicy;
import io.jmix.securityui.role.annotation.ScreenPolicy;

@Role(name = FullAccessRole.ROLE_NAME, code = FullAccessRole.ROLE_NAME)
public interface FullAccessRole {
    String ROLE_NAME = "system-full-access";

    @EntityPolicy(entityName = "*", actions = {EntityPolicyAction.ALL})
    @EntityAttributePolicy(entityName = "*", attributes = "*", actions = {
            EntityAttributePolicyAction.READ, EntityAttributePolicyAction.UPDATE})
    @ScreenPolicy(screenIds = "*")
    @MenuPolicy(menuIds = "*")
    @SpecificPolicy(resources = "*")
    void fullAccess();
}

File: jmix-templates/content/project/application-kotlin/src/main/kotlin/${project_rootPath}/security/FullAccessRole.kt
package ${project_rootPackage}.security

import io.jmix.security.model.EntityAttributePolicyAction
import io.jmix.security.model.EntityPolicyAction
import io.jmix.security.role.annotation.EntityAttributePolicy
import io.jmix.security.role.annotation.EntityPolicy
import io.jmix.security.role.annotation.ResourceRole
import io.jmix.security.role.annotation.SpecificPolicy
import io.jmix.securityui.role.annotation.MenuPolicy
import io.jmix.securityui.role.annotation.ScreenPolicy

@ResourceRole(name = "Full Access", code = FullAccessRole.CODE)
interface FullAccessRole {

    companion object {
        const val CODE = "system-full-access"
    }

    @EntityPolicy(entityName = "*", actions = [EntityPolicyAction.ALL])
    @EntityAttributePolicy(entityName = "*", attributes = ["*"], action = EntityAttributePolicyAction.MODIFY)
    @ScreenPolicy(screenIds = ["*"])
    @MenuPolicy(menuIds = ["*"])
    @SpecificPolicy(resources = ["*"])
    fun fullAccess()
}